network/load-balancer/reference-content/configuring-load-balancer.mdx

```diff
@@ -0,0 +1,137 @@
+
+When the Load Balancer initiates an HTTPS connection to a backend server, the backend server presents its certificate. 
+- When Verify Certificate is selected, the Load Balancer checks the backend server's certificate and only connects if it is valid.
+- When Verify Certificate is unselected, the Load Balancer does not check the backend server's certificate, connecting to the backend server regardless of the certificate's validity. 
+
+This setting mostly concerns those using self-signed certificates on the backend server/s.
+
+Note that the same Verify Certificate setting you select here will also be used for health checks, if you choose HTTPS for your [health check method](#choosing-a-health-check-method).
+
+### Choosing a proxy protocol
+
+Proxy protocol determines how connection information from the client (eg the client’s IP address) will be transferred through the Load Balancer and on to the destination backend server. This allows the backend server to know the client's address, whatever the upper layer protocol (HTTP/HTTPS/TCP). 
+
+* **No proxy protocol**:  Disable proxy protocol.
+* **Proxy protocol v1**: Version one (text format).
+* **Proxy protocol v2**: Version two (binary format).
+* **Proxy protocol v2-ssl**: Version two with SSL connection
+* **Proxy protocol v2-ssl-cn**:  Version two with SSL connection and common name information.
+
+### Choosing a health check method
+
+Load Balancers should only forward traffic to “healthy” backend servers. To monitor the health of a backend server, health checks regularly attempt to connect to backend servers using the defined protocol to ensure that servers are up and listening. Various protocols for health checks are available:
+
+* **HTTP**: When selecting HTTP heath checks, you are prompted to choose a **method** (`GET`, `POST` or `PUT`), the **URI** to be targeted, and the response **code** that signals a successful health check.
+* **HTTPS**: The same method and fields as for an HTTP health check apply, but data transmission is encrypted. If you chose HTTPS as the main protocol for the Load Balancer's backend, the Verify Certificate setting you chose there is inherited for health checks. If you chose a different protocol for the Load Balancer's backend, the backend server's certificate will **not** be verified during health checks. See the [dedicated Verify Certificate section](#choosing-a-verify-certificate-setting) for more information.
+* **MYSQL**: For MYSQL health checks, you are prompted to select the database username.
+* **PGSQL**: For PGSQL health checks, you are prompted to select the database username.
+* **LDAP**: No additional settings are required fr LDAP health checks.
+* **Redis**: No additional settings are required for REDIS health checks.
+* **TCP**: No additional settings are required for TCP health checks.
```

Comment:
TCP check description: http://docs.haproxy.org/2.6/configuration.html#4.2-option%20tcp-check

I believe here we could just say that the check consists in a connection attempt that either fails or succeeds.